lib/cosmos/tools/cmd_tlm_server/cmd_tlm_server_config.rb

```diff
@@ -130,23 +130,31 @@ def process_file(filename, recursive = false)
             parser.verify_num_parameters(2, nil, usage)
             interface_name = params[0].upcase
             raise parser.error("Interface '#{interface_name}' defined twice") if @interfaces[interface_name]
-            interface_class = Cosmos.require_class(params[1])
-            if params[2]
-              current_interface_or_router = interface_class.new(*params[2..-1])
-            else
-              current_interface_or_router = interface_class.new
+            begin
+              interface_class = Cosmos.require_class(params[1])
+              if params[2]
+                current_interface_or_router = interface_class.new(*params[2..-1])
+              else
+                current_interface_or_router = interface_class.new
+              end
+              current_type = :INTERFACE
+              current_interface_log_added = false
+              current_interface_or_router.packet_log_writer_pairs << @packet_log_writer_pairs['DEFAULT']
+              current_interface_or_router.name = interface_name
+              @interfaces[interface_name] = current_interface_or_router
+            rescue LoadError
```

Comment:
current_interface_or_router needs to be cleared here so that things will not work if there was a previous interface that is still in context.